### ISSUE TITLE
New request: wikiwel.com

### ISSUE THREAD

**Issue by hamoudak** [openzim/zim-requests#1164]
*Please use the following format for a ZIM creation request (and delete unnecessary information)*

- Website URL: https://wikiwel.com
- License: not copyrighted
- Desired ZIM Title: wikiwel
- Desired ZIM Description: Wikiwel is a non-profit organisation, created to share knowledge of traditional, natural healing methods and to create a bridge between Eastern (TCM, Ayurveda) and Western traditional medicine (Herbalism, Homeopathy,..) 

- Desired ZIM Icon –png (URL or attach one): https://wikiwel.com/wikihealing/favicon.ico
- Language (ISO 639-3): eng
- Is this a MediaWiki?: yes; powered by mediawiki

- Note: could it be with full resolution images if possible? it's a small website.

**Comment by RavanJAltaie** [comment]
Recipe created as per agreed with @Popolechien to check dev version of Mwoffliner
https://farm.openzim.org/recipes/wikiwel.com_en_all
I'll check the progress and update it here.

**Comment by benoit74** [comment]
Unfortunately as can been seen on https://wikiwel.com/wikihealing/index.php?title=Special:Version, this wiki does not have appropriate extensions available that [mwoffliner requires ](https://github.com/openzim/mwoffliner/wiki/API-end%E2%80%90points) to scrape it.

So not feasible for now, maybe someone wants to contact website owner to ask for installation of RestApi, VisualEditor or ActionParse extension (in order of preference, and one is supposed to be sufficient if I'm not mistaken)?

**Comment by hamoudak** [comment]
no you're right, I see this wik was made before (2018-10) last date; and I thought to get a better version of it, I didn't know about these requirements. anyway could anyone contact them in the near future.

**Comment by kelson42** [comment]
It seems wikiwel has removed his wiki... and therefore we should probably removed everything related to Wikiwel and close this issue.

**Comment by hamoudak** [comment]
yes, I have noticed that from long time ago. It was better than the website.

**Comment by benoit74** [comment]
I've deleted the recipe. No ZIM to delete since it never worked.
